actionpack/lib/action_dispatch/journey/route.rb

```diff
@@ -90,7 +90,7 @@ def ast
       end
 
       def requirements
-        # needed for rails `rails routes`
+        # Needed for `rails routes`.
```

Comment:
I think this comment should be moved outside the method body and describe the purpose of the method and it's primary use case in `rails routes`.